generator/api/src/main/java/io/fabric8/maven/generator/api/FromSelector.java

```diff
@@ -126,7 +134,15 @@ public boolean isJava11() {
     }
 
     private boolean isEleven(String s) {
-        return Configs.asInt(s) == 11;
+        boolean result = false;
+        try {
+            result = Configs.asInt(s) == 11;
+        } catch (NumberFormatException e) {
+            // The user has set a 1.x (eg, 1.8) value as target/release java version.
+            logger.info("%.1f target Java version detected", Configs.asFloat(s));
```

Comment:
If I were you, I would have done a `logger.debug`

Reply:
changes made